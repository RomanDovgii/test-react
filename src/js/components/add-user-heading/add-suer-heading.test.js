import React from "react";
import renderer from "react-test-renderer";
import AddUserHeading from "./add-user-heading";

it(
    `Should render correct AddUserHeading`,
    () => {
      const tree = renderer
      .create(
          <AddUserHeading/>
      )
      .toJSON();

      expect(tree).toMatchSnapshot();
    }
);
