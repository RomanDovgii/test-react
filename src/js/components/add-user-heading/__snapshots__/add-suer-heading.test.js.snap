// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correct AddUserHeading 1`] = `
<tr
  className="add-user__row"
>
  <th
    className="add-user__heading"
  >
    id
  </th>
  <th
    className="add-user__heading"
  >
    firstName
  </th>
  <th
    className="add-user__heading"
  >
    lastName
  </th>
  <th
    className="add-user__heading"
  >
    email
  </th>
  <th
    className="add-user__heading"
  >
    phone
  </th>
</tr>
`;
